Framework/DataHandling/src/LoadBBY.cpp

```diff
@@ -584,9 +580,9 @@ LoadBBY::createInstrument(ANSTO::Tar::File &tarFile,
 
   // patching
   std::string logContent;
-  for (auto itr = files.begin(); itr != files.end(); ++itr)
-    if (itr->compare("History.log") == 0) {
-      tarFile.select(itr->c_str());
+  for (const auto &file : files)
+    if (file.compare("History.log") == 0) {
+      tarFile.select(file.c_str());
```

Comment:
I think this is the same as the previous but even shorter because there is no special compare function:
```
const auto file_it = std::find(files.cbegin(), files.cend(), "History.log");
if (file_it != files.cend()) {
  tarFile.select(file_it->c_str());
}
```